DSA/4.1 Stacks/01. Introduction/3.Introduction_to_Stacks_video.md

```diff
@@ -0,0 +1,15 @@
+## Introduction to stacks:
```

Comment:
Kindly remove this file of stack and remain that file only which was for sorting a stack